static/js/alert.js

```diff
@@ -66,7 +66,34 @@ let mapIndexToAlertState=new Map([
 const alertForm =$('#alert-form');
 
 $(document).ready(function () {
-
+    $('#evaluate-every').tooltip({
+        title: 'How often the alert will be evaluated to see if it fires',
+        placement: 'top',
+        trigger: 'manual'
+    });
+    const urlParams = new URLSearchParams(window.location.search);
+    $("#alert-rule-name").val(urlParams.get('alertRule_name'));
+    $('#evaluate-for').tooltip({
+        title: 'Evaluate for interval must be greater than evaluation interval',
+        placement: 'top',
+        trigger: 'manual' 
+    });
+    $('#evaluate-for').on('input', function () {
+        let evaluateEveryValue = parseInt($('#evaluate-every').val());
+        let evaluateForValue = parseInt($(this).val());
+        let submitbtn=$("#save-alert-btn");
+
+        if (evaluateForValue > evaluateEveryValue) {
```

Comment:
1. The condition should be `>=` .
2. Make the default value for `for evaluation` as `0`.
3. As you update the `evaluate every` value and it becomes greater than the `for` value the tooptip should get remove automatically.
<img width="494" alt="image" src="https://github.com/siglens/siglens/assets/71771131/6dcd57cd-adca-4838-b75a-d94cd6f0e584">